### ISSUE TITLE
Improve docs: why do we ignore almost all the FP jargon in this project

### ISSUE THREAD

**Issue by sobolevn** [dry-python/returns#451]
- [ ] Describe our decision
- [ ] Add a link to: https://github.com/jmesyou/functional-programming-jargon.py
- [ ] Improve https://github.com/jmesyou/functional-programming-jargon.py with `returns` links and examples

**Comment by thepabloaguilar** [comment]
Can FP users understand what is "a thing" without jargon? Can they correlate "a thing" to a jargon?
If the answers to these questions are "yes" I think we can "ignore" the FP jargon to be more inclusive with people that do not understand it!

*On the other hand, we have to get these people familiar with FP jargon to understand what they are using.*

**Comment by thepabloaguilar** [comment]
What's the final decision here? Will we get out of FP jargons?

**Comment by sobolevn** [comment]
Yes, we don't use it. (As a matter of fact, we already don't use it).

The history behind this is really interesting. The first name of this project was `dry-monads`: https://github.com/dry-python/returns/tree/0.1.0

But, then I got a lot of uneducated feedback. It was hard to manage. So, instead we went to make this concept hidden.
And it worked! 

Basically, we came up with our own terms that do mean the similar things. And we should stick to them.

**Comment by thepabloaguilar** [comment]
So, the first item was completed I guess.
> * [X]  Describe our decision

Answer:

> then I got a lot of uneducated feedback. It was hard to manage. So, instead we went to make this concept hidden.
And it worked!
> Basically, we came up with our own terms that do mean the similar things. And we should stick to them.

---

Where we have to add the link? On the first page?
> * [ ]  Add a link to: https://github.com/jmesyou/functional-programming-jargon.py

What exactly we have to improve? 
> * [ ]  Improve https://github.com/jmesyou/functional-programming-jargon.py with `returns` links and examples

**Comment by sobolevn** [comment]
> What exactly we have to improve?

There are a lot of concepts that are poorly illustrated in the original article: https://github.com/jmesyou/functional-programming-jargon.py 

I guess we can improve almost every point in there 🙂

**Comment by thepabloaguilar** [comment]
If we create a repository inside `dry-python` it's easier to update it, you've opened a PR there 23 days ago and so far you haven't got any response!

**Comment by sobolevn** [comment]
Great idea!

**Comment by sobolevn** [comment]
New repo: https://github.com/dry-python/functional-jargon-python

**Comment by thepabloaguilar** [comment]
How do you want to do this??
Maybe copy-paste the structure from [https://github.com/jmesyou/functional-programming-jargon.py](https://github.com/jmesyou/functional-programming-jargon.py) without the content of each topic and we can progressively be adding the content by ourselves?

**Comment by sobolevn** [comment]
Sounds like a plan 👍

**Comment by thepabloaguilar** [comment]
Can you initialize the repo, please? Because I can't fork if it's empty 😞

**Comment by thepabloaguilar** [comment]
Just a simple README or a LICENSE hahaha

**Comment by sobolevn** [comment]
@thepabloaguilar you can use the original repo, don't need to fork it. Do you have the right permissions?

**Comment by thepabloaguilar** [comment]
Done, I've pushed the content!!
I decide to keep some texts just to help in some way, but every topic is marked as `TODO`
Now we can start filling the gaps hahaha

**Comment by sobolevn** [comment]
Awesome! 👍
